app/assets/javascripts/global-bar-init.js

```diff
@@ -36,7 +36,11 @@ var globalBarInit = {
       paths.push(ctaLink.getAttribute('href'))
     }
 
-    return new RegExp(paths.join("|")).test(window.location.pathname)
+    var regexForPaths = paths.map(function(path) {
+      return "^" + path + "$";
```

Comment:
I've got a feeling that the other paths in this might be prefix things - `/done` goes before a bunch of pages and I think the transition checker has lots of sub pages too.  Maybe if we drop the trailing `$`?

Reply:
Good point.

If I drop the trailing `$` then this would match `/guidance/coronavirus-covid-19-information-for-the-public`.

There are two approaches we can take:
1. Two separate lists, one which has a wildcard match and another which is an exact match. Before we create the regex, we would merge it together.
2. Put the regex directly in the list so that we can specify the paths as we want.

```javascript

var paths = [
      "^/done",
      "^/transition-check$",
      "^/coronavirus$"
    ]

```

Reply:
@sihugh - updated